### PR TITLE
Improving hot reload dependencies provider

### PR DESCRIPTION
**Pull request by munerf**

### CODE REVIEW

File: ReactViewControl/IViewModule.cs [hunk 1]
```diff
@@ -22,7 +23,7 @@ public interface IViewModule {
 
         KeyValuePair<string, object>[] PropertiesValues { get; }
 
-        void Bind(IFrame frame, IChildViewHost host = null);
+        void Bind(IFrame frame, IChildViewHost host = null, Uri devServerUri = null);
```
Comment (alvesmiguel1) [inline ReactViewControl/IViewModule.cs:26]: Hum.. This is something internal, we shouldn't have to expose this to the public API

Comment (munerf) [reply ReactViewControl/IViewModule.cs:26]: you suggested this change some time ago :S

Comment (munerf) [reply ReactViewControl/IViewModule.cs:26]: any alternative suggestion?

**Comment by joaompneves** [comment]
Moved here: https://github.com/OutSystems/WebView-CefSharp/tree/hot-reload-fixes

**Pull request closed without merging**